File: 프로그래머스/unrated/181856. 배열 비교하기/배열 비교하기.py
def solution(arr1, arr2):
    len_arr1, len_arr2 = len(arr1), len(arr2)  # 두 배열의 길이를 구합니다.

    # 두 배열의 길이를 비교합니다.
    if len_arr1 > len_arr2:
        return 1
    elif len_arr1 < len_arr2:
        return -1

    # 두 배열의 길이가 같은 경우, 배열의 원소들의 합을 비교합니다.
    sum_arr1, sum_arr2 = sum(arr1), sum(arr2)
    if sum_arr1 > sum_arr2:
        return 1
    elif sum_arr1 < sum_arr2:
        return -1

    # 두 배열의 길이와 원소들의 합이 모두 같은 경우
    return 0

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.java
import java.util.Arrays;

class Solution {
    public int solution(int[] arr1, int[] arr2) {
        int answer = 0;
        int len1 = arr1.length;
        int len2 = arr2.length;
        if(len1 == len2) {
            int sum1 = Arrays.stream(arr1).sum();
            int sum2 = Arrays.stream(arr2).sum();
            if(sum1>sum2){
                answer=1;
            }else if(sum1<sum2){
                answer=-1;
            }else{
                answer=0;
            }
        }else if(len1 > len2){
            answer=1;
        }else{
            answer=-1;
        }
        return answer;
    }
}